code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1351,7 +1351,9 @@
 /datum/reagent/medicine/changelingadrenaline/on_mob_life(mob/living/carbon/metabolizer, seconds_per_tick, times_fired)
 	. = ..()
 	metabolizer.AdjustAllImmobility(-20 * REM * seconds_per_tick)
-	if(metabolizer.adjustStaminaLoss(-10 * REM * seconds_per_tick, updating_stamina = FALSE))
+	// If stamina critted, brings them out of it; otherwise, heals (quite a bit) of stamina
+	// damage without totally negating the slowness of having taken stamina damage
+	if(metabolizer.adjustStaminaLoss(metabolizer.getStaminaLoss() > 100 ? -21 : -10 * REM * seconds_per_tick, updating_stamina = FALSE))
```

Comment:
This is just goofy, you might as well just make them immune to stamcrit at this point

Reply:
This is to make them vulnerable to stamina slowdown. You slow if you have more than 40 stam damage. Having it a static 20 or 21 seemed too much, but the situation of being stunlocked by thugs with baton was the main killer of lings. If they're an instant round removal threat, they should be more threatening, without being immune to stamina slowdown.

Reply:
I didn't say make them immune to stamina slowdown, I said make them immune to stamcrit